src/pseudopeople/noise_functions.py

```diff
@@ -43,7 +43,9 @@ def omit_rows(
     """
     dataset.data = dataset.data.loc[dataset.data.index.difference(to_noise_index)]
     # TODO: Mic-4875 add update_missingness method to Dataset
-    dataset.missingness.loc[to_noise_index, :] = True
+    dataset.missingness = dataset.missingness.loc[
```

Comment:
Not sure I understand. Previoiusly you set all values at `to_noise_index` to True but keeps the rest of the index intact. This now totally overwrites `dataset.missingness` to just be the existing values of only the not-`to_noise_index`. Is this the intended behavior?

Reply:
Yes. Dataset.missingness is a boolean dataframe that should indicated whether or not Dataset.data has null values. So the behavior before (which is now being overwritten) was preserving the original shape of the dataframe but when we dropped rows, we also want to drop them in the missingness attribute so the two dataframes have the same shape and are in sync.